app/assets/stylesheets/oregon_digital/_dashboard_sidebar.scss

```diff
@@ -0,0 +1,3 @@
+.sidebar-header {
```

Comment:
This doesn't seem necessary and makes the sidebar messy. The whole backgrounds aren't change, just a thin strip around the text. With the text being white, it adds enough contrast to the entire sidebar heading and the other text used to be white already (and was passing contrast). Lets keep the CSS simpler and remove the classes below